components/src/molecules/cookie-message/CookieMessage.js

```diff
@@ -8,34 +8,32 @@ export default {
     AButton
   },
   props: {
-    text: {
-      type: String,
-      required: true
-    },
-    dataType: {
-      type: String,
-      required: true
-    },
-    closeIcon: {
-      type: String,
-      default: null
-    },
+    /**
+     * Text for close button
+     */
     closeButton: {
       type: String,
       default: null
     },
+    /**
+     * Aria label for container
+     */
     ariaLabel: {
       type: String,
-      default: 'Cookie policy message'
+      default: null
     },
+    /**
+     * Aria label for close button
+     */
     closeAriaLabel: {
       type: String,
-      default: 'Close cookie message'
+      default: null
     }
   },
   data: () => {
     return {
-      isOpen: false
+      isOpen: false,
+      dataType: 'cookie'
```

Comment:
Sorry, I was wrong about moving this to data, I thought it's a reactive data.
Could you move it back to prop and rename to something like `storage-name`? With default `cookie`